src/pykeen/nn/representation.py

```diff
@@ -1970,3 +2095,68 @@ def _plain_forward(self, indices: LongTensor | None = None) -> FloatTensor:  # n
         return einsum(
             self.eq, *(base(indices) for indices, base in zip(assignment.unbind(dim=-1), self.bases, strict=False))
         ).view(*assignment.shape[:-1], *self.shape)
+
+
+@parse_docdata
+class FeatureEnrichedEmbedding(CombinedRepresentation):
```

Comment:
Maybe it makes sense to separate "core" components and those "factory-like" components, which are essentially just a short-cut for a specific instantiation of parameters, at some point.

Reply:
that's a good idea. We could already start that here. Maybe put these in a module called `pykeen.nn.factory`, `pykeen.nn.component`, or `pykeen.nn.meta` (other suggestions welcome)